projects/packages/search/src/dashboard/components/upsell-nudge/index.jsx

```diff
@@ -14,10 +14,10 @@ export default function InstantSearchUpsellNudge( props = { upgrade: true } ) {
 		<a className="jp-instant-search-upsell-nudge jp-search-dashboard-cut" href={ props.href }>
 			<span>
 				{ __(
-					'Offer instant search results to your visitors as soon as they start typing. ',
+					'Offer instant search results to your visitors as soon as they start typing.',
 					'jetpack-search-pkg'
 				) }
-			</span>
+			</span>{ ' ' }
```

Comment:
Should we put the space inside the `span` element to preserve the original behavior?

Reply:
🤷 No idea. In practice CSS currently sets `display: block` on these spans so the whitespace gets ignored either way.

Reply:
Before:
![before](https://github.com/user-attachments/assets/2ca70171-3887-49d4-aaf6-c6b5e616e2d7)

After:
![after](https://github.com/user-attachments/assets/537c6205-640d-4996-a398-55d018401a84)